xtos/include/sof/lib/dma.h

```diff
@@ -198,6 +198,8 @@ struct dma_ops {
 	int (*get_data_size)(struct dma_chan_data *channel, uint32_t *avail,
 			     uint32_t *free);
 
+	int (*clear_data_copied)(struct dma_chan_data *channel);
```

Comment:
@dbaluta  one question here, there is a file posix/include/sof/lib/dma.h, should we need to implement it?

Reply:
for us, no. We dont build for posix